packages/javascript-mutator/src/JavaScriptMutator.ts

```diff
@@ -25,7 +25,7 @@ export class JavaScriptMutator implements Mutator {
         this.mutators.forEach(mutator => {
           const mutatedNodes = mutator.mutate(node, copy);
 
-          if (mutatedNodes) {
+          if (mutatedNodes.length) {
```

Comment:
With this improvement, we don't need to check on anything, right? We can just remove this if all together ❤

Reply:
even if this is an array, we should check if the length is > 0 to ensure we won't run unnecessarily some part of code

Reply:
take note that array.push is quite an expensive function so we shouldn't overdo it. Also, the spread operator is quite a performance killer (even if both are just taking small parts of ms, we ms to ms and you get s)

Reply:
Readability is more important than performance. Especially here, it won't make a difference. The push function will be called without arguments, which should be fast to execute. 

In any case, we shouldn't care about it. Generating mutants in memory takes less than ~0.01% of the total mutation testing time, right? If we can make it more readable and maintainable by removing this code branch, we should IMHO.

Reply:
@nicojs and what about leaving these 'push' but at least removing 'void' type from mutators?

Reply:
Do you mean removing the `void` from the return type of `mutate` in the different mutators? Yeah, I'm all for that! Cleans up the code nicely 💐

Reply:
It will be done with easy (soon. I am sick as hell, but probably I'll manage to do it tomorrow)

Reply:
Wow man.. just get well, don't worry about it. @simondel and I carved out some time in our schedules to spent on Stryker next week. We'll be merging all PR then as well as fix some issues ourselves. 🎉

Reply:
@nicojs its done btw XD